### ISSUE TITLE
Videos behaving strangely on the homepage 

### ISSUE THREAD

**Issue by para2022** [WWBN/AVideo#8597]
Videos are appearing blank on upload. I see it happen with encoder uploads and direct upload. A while later the thumbnail appears as normal. I didn't see this before today. Is this a cache issue? I also updated my encoders today. 

Also randomly an upload a video gets stuck in the recent uploads. For instance it will stay as the most recent upload even though other videos are a lot more recent than that. See the bottom image.

![Screenshot 2023-11-01 at 17 38 16](https://github.com/WWBN/AVideo/assets/107706817/f7660374-953d-4478-8f34-252e1a8145ff)

![Screenshot 2023-11-01 at 17 43 07](https://github.com/WWBN/AVideo/assets/107706817/744d202c-2f76-4668-91fc-4c28b11ed809)

![Screenshot 2023-11-01 at 16 36 38](https://github.com/WWBN/AVideo/assets/107706817/52eb70e2-0ef8-47ab-9fe2-c2b12cf97017)

**Comment by para2022** [comment]
[client 2607:fb90:e2d5:425b:b95c:2354:b7fd:bee3:34402] AH01071: Got error '"args":["video_231101155006_v7733"]},{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/plugin\\/MP4ThumbsAndGif\\/MP4ThumbsAndGif.php(18) : eval()'d code(1) : eval()'d code","line":410,"function":"getImageInDuration","class":"MP4ThumbsAndGif","type":"::","args":[53520,"jpg"]},{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/plugin\\/AVideoPlugin.php","line":890,"function":"onUploadIsDone","class":"MP4ThumbsAndGif","object":{},"type":"->","args":[53520]},{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/view\\/mini-upload-form\\/upload.php","line":137,"function":"onUploadIsDone","class":"AVideoPlugin","type":"::","args":[53520]}] SCRIPT_NAME: /view/mini-upload-form/upload.php; PHP message: AVideoLog::DEBUG: deleteMediaSessionPosters (/home/ourtube/public_html/ourtube/videos/video_231101155006_v7733/video_231101155006_v7733_96.jpg) unlink line=10665 SCRIPT_NAME: /view/mini-upload-form/upload.php; PHP message: AVideoLog::DEBUG: deleteMediaSessionPosters (/home/ourtube/public_html/ourtube/videos/video_231101155006_v7733/video_231101155006_v7733_128.jpg) unlink line=10665 SCRIPT_NAME: /view/mini-upload-form/upload.php; PHP message: AVideoLog::DEBUG: deleteMediaSessionPosters (/home/ourtube/public_html/ourtube/videos/video_231101155006_v7733/video_231101155006_v7733_192.jpg) unlink line=10665 SCRIPT_NAME: /view/mini-upload-form/upload.php; PHP message: AVideoLog::DEBUG: deleteMediaSessionPosters (/home/ourtube/public_html/ourtube/videos/video_231101155006_v7733/video_231101155006_v7733_256.jpg) unlink line=10665 SCRIPT_NAME: /view/mini-upload-form/upload.php; PHP message: AVideoLog::DEBUG: deleteMediaSessionPosters (/home/ourtube/public_html/ourtube/videos/video_231101155006_v7733/video_231101155006_v7733_384.jpg) unlink line=10665 SCRIPT_NAME: /view/mini-upload-form/upload.php; PHP message: AVideoLog::DEBUG: deleteMediaSessionPosters (/home/ourtube/public_html/ourtube/videos/video_231101155006_v7733/video_231101155006_v7733_512.jpg) unlink line=10665 SCRIPT_NAME: /view/mini-upload-form/upload.php; PHP message: AVideoLog::DEBUG: deleteThumbs [{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/plugin\\/MP4ThumbsAndGif\\/MP4ThumbsAndGif.php(18) : eval()'d code(1) : eval()'d code","line":300,"function":"deleteThumbs","class":"Video","type":"::","args":["video_231101155006_v7733"]},{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/plugin\\/MP4ThumbsAndGif\\/MP4ThumbsAndGif.php(18) : eval()'d code(1) : eval()'d code","line":410,"function":"getImageInDuration","class":"MP4ThumbsAndGif","type":"::","args":[53520,"jpg"]},{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/plugin\\/AVideoPlugin.php","line":890,"function":"onUploadIsDone","class":"MP4ThumbsAndGif","object":{},"type":"->","args":[53520]},{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/view\\/mini-upload-form\\/upload.php","line":137,"function":"onUploadIsDone","class":"AVideoPlugin","type":"::","args":[53520]}] SCRIPT_NAME: /view/mini-upload-form/upload.php; PHP message: AVideoLog::DEBUG: Video:deleteThumbs(/home/ourtube/public_html/ourtube/videos/video_231101155006_v7733/video_231101155006_v7733_thumbsSprit.jpg) unlink line=5520 SCRIPT_NAME: /view/mini-upload-form/upload.php; PHP message: AVideoLog::DEBUG: Video:deleteThumbs(video_231101155006_v7733) [{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/objects\\/video.php","line":5573,"function":"deleteThumbs","class":"Video","type":"::","args":["video_231101155006_v7733",true]},{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/plugin\\/MP4ThumbsAndGif\\/MP4ThumbsAndGif.php(18) : eval()'d code(1) : eval()'d code","line":301,"function":"clearCache","class":"Video","type":"::","args":[53520,true]},{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/plugin\\/MP4ThumbsAndGif\\/MP4ThumbsAndGif.php(18) : eval()'d code(1) : eval()'d code","line":410,"function":"getImageInDuration","class":"MP4ThumbsAndGif","type":"::","args":[53520,"jpg"]},{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/plugin\\/AVideoPlugin.php","line":890,"function":"onUploadIsDone","class":"MP4ThumbsAndGif","object":{},"type":"->","args":[53520]},{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/view\\/mini-upload-form\\/upload.php","line":137,"function":"onUploadIsDone","cla...; PHP message: AVideoLog::DEBUG: deleteMediaSessionPosters (/home/ourtube/public_html/ourtube/videos/video_231101155006_v7733/video_231101155006_v7733_96.jpg) unlink line=10665 SCRIPT_NAME: /view/mini-upload-form/upload.php; PHP message: AVideoLog::DEBUG: deleteMediaSessionPosters (/home/ourtube/public_html/ourtube/videos/video_231101155006_v7733/video_231101155006_v7733_128.jpg) unlink line=10665 SCRIPT_NAME: /view/mini-upload-form/upload.php; PHP message: AVideoLog::DEBUG: deleteMediaSessionPosters (/home/ourtube/public_html/ourtube/videos/video_231101155006_v7733/video_231101155006_v7733_192.jpg) unlink line=10665 SCRIPT_NAME: /view/mini-upload-form/upload.php; PHP message: AVideoLog::DEBUG: deleteMediaSessionPosters (/home/ourtube/public_html/ourtube/videos/video_231101155006_v7733/video_231101155006_v7733_256.jpg) unlink line=10665 SCRIPT_NAME: /view/mini-upload-form/upload.php; PHP message: AVideoLog::DEBUG: deleteMediaSessionPosters (/home/ourtube/public_html/ourtube/videos/video_231101155006_v7733/video_231101155006_v7733_384.jpg) unlink line=10665 SCRIPT_NAME: /view/mini-upload-form/upload.php; PHP message: AVideoLog::DEBUG: deleteMediaSessionPosters (/home/ourtube/public_html/ourtube/videos/video_231101155006_v7733/video_231101155006_v7733_512.jpg) unlink line=10665 SCRIPT_NAME: /view/mini-upload-form/upload.php; PHP message: AVideoLog::DEBUG: deleteThumbs [{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/objects\\/video.php","line":5573,"function":"deleteThumbs","class":"Video","type":"::","args":["video_231101155006_v7733",true]},{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/plugin\\/MP4ThumbsAndGif\\/MP4ThumbsAndGif.php(18) : eval()'d code(1) : eval()'d code","line":301,"function":"clearCache","class":"Video","type":"::","args":[53520,true]},{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/plugin\\/MP4ThumbsAndGif\\/MP4ThumbsAndGif.php(18) : eval()'d code(1) : eval()'d code","line":410,"function":"getImageInDuration","class":"MP4ThumbsAndGif","type":"::","args":[53520,"jpg"]},{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/plugin\\/AVideoPlugin.php","line":890,"function":"onUploadIsDone","class":"MP4ThumbsAndGif","object":{},"type":"->","args":[53520]},{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/view\\/mini-upload-form\\/upload.php","line":137,"function":"onUploadIsDone","class":"AVideoPlugin","type":"::","...; PHP message: AVideoLog::DEBUG: clearCache starts 1 SCRIPT_NAME: /view/mini-upload-form/upload.php; PHP message: AVideoLog::DEBUG: rrmdir: do not delete videos or cache folder /home/ourtube/public_html/ourtube/videos/cache/ SCRIPT_NAME: /view/mini-upload-form/upload.php; PHP message: AVideoLog::DEBUG: clearCache end in 0.0021941661834717 seconds SCRIPT_NAME: /view/mini-upload-form/upload.php; PHP message: AVideoLog::DEBUG: getVideosURL_V2(video_231101155006_v7733) Recreate cache requested [{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/objects\\/functions.php","line":1500,"function":"getVideosURL_V2"},{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/plugin\\/MP4ThumbsAndGif\\/MP4ThumbsAndGif.php(18) : eval()'d code(1) : eval()'d code","line":348,"function":"getAudioOrVideoURLOnly"},{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/plugin\\/MP4ThumbsAndGif\\/MP4ThumbsAndGif.php(18) : eval()'d code(1) : eval()'d code","line":241,"function":"getFirstVideoURL","class":"MP4ThumbsAndGif","type":"::"},{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/plugin\\/MP4ThumbsAndGif\\/MP4ThumbsAndGif.php(18) : eval()'d code(1) : eval()'d code","line":411,"function":"getImageInDuration","class":"MP4ThumbsAndGif","type":"::"},{"file":"\\/home\\/ourtube\\/public_html\\/ourtube\\/plugin\\/AVideoPlugin.php","line":890,"function":"onUploadIsDone","class":"MP4ThumbsAndGif","type":"->"

**Comment by DanielnetoDotCom** [comment]
what happens if you clear the cache?

do you have something in the sort section?

![chrome-capture-2023-10-1 (2)](https://github.com/WWBN/AVideo/assets/18009293/081121b8-81cf-44a3-a7ea-d36b6ebffc7c)

**Comment by para2022** [comment]
I just had one stuck for 3 hours and it was in the sort section some how. When i deleted it from sort listings it went back to how it should display on the videos grid. Why are random videos going into the sort section?

**Comment by para2022** [comment]
happened again this morning. I have to keep removing random videos out of the sort for them to go back to normal

**Comment by para2022** [comment]
Yes i am up to date. yep its wierd because it will be any random video that ends up there and it is always just the 1, not more. The video will stay at the top until it is removed from sort list. Their is a bug also in the load more. If i click on load more and load a page, then that page is different from if i was to click on say 'Trending'. Their will be videos missing when i click on load more and if i click on trending on the homepage then the  missing vids from load more are then there.

**Comment by para2022** [comment]
First image is the load more list and second image is clicking on that page and as you can see they are different. I wonder if this is linked the reported issue?

![Screenshot 2023-11-02 at 12 37 09](https://github.com/WWBN/AVideo/assets/107706817/4e8670bc-2967-42cd-af85-0221d796f2c1)

![Screenshot 2023-11-02 at 12 37 43](https://github.com/WWBN/AVideo/assets/107706817/e47d7a77-2f7c-4ecc-b774-0dc1fe3aa138)

**Comment by DanielnetoDotCom** [comment]
but if you remove the sort video it is back to normal right?

**Comment by para2022** [comment]
yea

**Comment by para2022** [comment]
If i remove the sort the video goes back in place. But it does not fix the layout in the images about as they are still different.

**Comment by DanielnetoDotCom** [comment]
I do not understand what you mean the images are different

**Comment by para2022** [comment]
If you look at both images in row 3 they are different. The top image is me pressing 'load more' on the home page for most recent uploads. And clicking the most recent link on the homepage loads different images as shown in both images 'row 3'

**Comment by DanielnetoDotCom** [comment]
Please check it now

**Comment by para2022** [comment]
The load more on homepage doesn't work now so i cannot see if its fixed. You press load more and the videos dont load and the 'load more' bar vanishes. It only works on suggested at the top of the page and anything else on the homepage, the 'load more' bar now misbehaves.

**Comment by para2022** [comment]
If i clear the cache the load more works on all homepage categories. A short while later the bug occurs again in the load more and it stops working correctly.

**Comment by DanielnetoDotCom** [comment]
I cannot reproduce it here. when the issue happens please check your logs

**Comment by para2022** [comment]
Nothing happens in logs

**Comment by para2022** [comment]
Developers tools reports this:

infinite-scroll.pkgd.min.js:11 
        
        
       GET https://ourtube.co.uk/page/2/2/?rowCount=12&isInfiniteScroll=1&showOnly=dateAddedOrder&current=1 404 (Not Found)
i.loadNextPage @ infinite-scroll.pkgd.min.js:11
onClick @ infinite-scroll.pkgd.min.js:11
(index):10550 infiniteScroll Could not load: https://ourtube.co.uk/page/2/2/?rowCount=12&isInfiniteScroll=1&showOnly=dateAddedOrder&current=1.

**Comment by para2022** [comment]
The sort problem also exists still, i woke up to 7 stuck in the sort list.

**Comment by DanielnetoDotCom** [comment]
how can I reproduce this error?

(index):10550 infiniteScroll Could not load: https://ourtube.co.uk/page/2/2/?rowCount=12&isInfiniteScroll=1&showOnly=dateAddedOrder&current=1.

**Comment by para2022** [comment]
www.ourtube.co.uk and click on the load more on the site. You will see the suggested 'load more' at the top of the homepage works and anything below that does not. You introduced the bug yesterday. It happened directly after that update.

**Comment by DanielnetoDotCom** [comment]
Can you reproduce that here?

https://demo.avideo.com/

did you use git pull to update?

**Comment by para2022** [comment]
Yes because it happens after a while. I just cleared out my tmp dir. It worked then whilst I was logged out. I logged in and then the problem came back and then persists even when logged out again. You will probs need to keep refereshing the page. Im using brave. This doesnt happen on MAC safari. It happens on firefox also. Just loaded firefox the load more worked first time, i refreshed then it was broken again.

**Comment by para2022** [comment]
cache problem! I cleared the cache then it started working again, but only for a short while. After a few page refreshes the problem returns. I clear cache and it works again. On and on the problem cycles.

**Comment by DanielnetoDotCom** [comment]
Can you reproduce on demo site?

**Comment by DanielnetoDotCom** [comment]
Admin
123

**Comment by para2022** [comment]
thanks

**Comment by para2022** [comment]
It seems ok on the demo site. Which does not mean this isnt an issue. I already said it happened exactly at the same time you told me to update the site. I used the load more all the time up until then. I Did the update and it was immediately broken so it is blatently a bug in the script/update. Never happened before that. Dev tools directly reports an error in the load more.

**Comment by DanielnetoDotCom** [comment]
If I cannot reproduce I cannot help you

**Comment by DanielnetoDotCom** [comment]
check if there is any outdated plugin

**Comment by para2022** [comment]
They are all up to date. I check for updates daily and check the plugins also. The same problem is also happening on my development server, ive just checked that site too. I have swapped themes and on both sites also its not that either.

**Comment by para2022** [comment]
The development site was not up to date. It worked fine and there was no load more bug there so I did a git pull and there appears the bug exact same as the other site.

**Comment by para2022** [comment]
Dan is it safe to download an up to date zip package of all files and overwrite all my files and see if it makes a difference?

**Comment by DanielnetoDotCom** [comment]
the correct way to update is to make a git pull. 

I do not like the zip upload

zip file is only for plugins

**Comment by para2022** [comment]
Sorry mate i actually meant I will do a git pull and rebuild the site from fresh files, obviously protecting the videos dir with config and files in. I will rebuild all plugins from scratch also. I will see how i get on after doing this.

**Comment by para2022** [comment]
reinstalled the site with all new files. No plugins are turned on. Still the exact same problem

![Screenshot 2023-11-07 at 21 38 36](https://github.com/WWBN/AVideo/assets/107706817/98ef221c-2ce5-40cc-b765-34dd7540dd6f)

**Comment by para2022** [comment]
[07-Nov-2023 21:37:12 Europe/London] AVideoLog::ERROR: Time: 2.091s   | Limit: 0.2s | Location: path/to/my/streamer/site/view/index.php Line 131 [navbar.php] IP=2a02:c7c:5f98:6500:6d74:933e:84:90f1 backtrace=[{"file":"\/home\/myMySQLUser\/public_html\/myMySQLUser\/view\/include\/navbar.php","line":131,"function":"TimeLogEnd","args":["navbar.php",131,0.2]},{"file":"\/home\/myMySQLUser\/public_html\/myMySQLUser\/objects\/Page.php","line":99,"args":["\/home\/myMySQLUser\/public_html\/myMySQLUser\/view\/include\/navbar.php"],"function":"include"},{"file":"\/home\/myMySQLUser\/public_html\/myMySQLUser\/objects\/Page.php","line":137,"function":"getNavBar","class":"Page","object":{},"type":"->","args":[]},{"file":"\/home\/myMySQLUser\/public_html\/myMySQLUser\/objects\/Page.php","line":148,"function":"getContent","class":"Page","object":{},"type":"->","args":[]},{"file":"\/home\/myMySQLUser\/public_html\/myMySQLUser\/objects\/Page.php","line":157,"function":"getPage","class":"Page","object":{},"type":"->","args":[]},{"file":"\/home\/myMySQLUser\/public_html\/myMySQLUser\/plugin\/Gallery\/view\/modeGallery.php","line":27,"function":"print","class":"Page","object":{},"type":"->","args":[]},{"file":"\/home\/myMySQLUser\/public_html\/myMySQLUser\/view\/index_firstPage.php","line":58,"args":["\/home\/myMySQLUser\/public_html\/myMySQLUser\/plugin\/Gallery\/view\/modeGallery.php"],"function":"require"},{"file":"\/home\/myMySQLUser\/public_html\/myMySQLUser\/view\/index.php","line":20,"args":["\/home\/myMySQLUser\/public_html\/myMySQLUser\/view\/index_firstPage.php"],"function":"include"}] SCRIPT_NAME: /view/index.php

**Comment by para2022** [comment]
Ive found the bug and this was beyond painstaking. It is in the gallery plugin which is why it is not affecting your site and why others have probably not reported it. If i turn off the big video (i do not use it) then the bug appears. If i put the big video back then the 'load more' on homepage works properly. I have tried this many times and it is indeed the cause of this issue. I have had to strip my site down to a fresh install to find the root cause.

**Comment by para2022** [comment]
Also suggested videos are not affected by this bug, and as the big video and suggested work together that could have been a clue to the problem

**Comment by DanielnetoDotCom** [comment]
thanks for finding some clues. but I disabled the big video in the demo site and I still could not reproduce it.

**Comment by para2022** [comment]
I dont get it at all. The sort issue is also still there. Its a brand new install. I woke up to videos in the sort area again. Like i said if i turn the big video on the load more works if its off it will not. Makes no sense whatsoever and ive tried everything imaginable other than rebooting the server. I have no errors anywhere in regards to this other than the devtools error when it happens. :(

**Comment by DanielnetoDotCom** [comment]
How can I reproduce it on your site? 

I went there and I could not reproduce it either

**Comment by para2022** [comment]
If you use brave or firefox and go to the mainpage and then click on the trending and whats new 'load more' bar. If you click the bar it may work the first time, then refresh the page a few times and try again. You will noticed the 'featured' up top always works. The bug does not affect whatever is first (at the top) of the page. It is any group below that that doesnt work. I really appreciate your help on this as its driving me nuts. Thanks.

**Comment by para2022** [comment]
Just to add to this. If you go into the page i.e. by clicking on the 'whats new' label on the homepage, the actually label to bring up the whats new page. If you get those pages up, the load more bar at the bottom of the page always works. It only affects the homepage this bug. Here is a quick link to it https://ourtube.co.uk/?showOnly=dateAddedOrder those pages are not affected.

**Comment by para2022** [comment]
Is this relevant. Devtools complains about this link not working. https://ourtube.co.uk/page/2/2/?rowCount=12&isInfiniteScroll=1&showOnly=TrendingOrder&current=1. Error: Not Found

I have edited that link that is broken and removed one of the /2/ from the link, see below. If i do that then the link works. Is it generating bad links?

https://ourtube.co.uk/page/2/?rowCount=12&isInfiniteScroll=1&showOnly=TrendingOrder&current=1

**Comment by DanielnetoDotCom** [comment]
I tried so many times to reproduce it in this page and it still working for me

**Comment by para2022** [comment]
i mentioned in a comment above that it does not affect those pages. The bug is on the homepage 'load more' system only. If you press 'load more' nothing happens and then the 'load more' bar vanishes

**Comment by DanielnetoDotCom** [comment]
I did press the load more and it works for me

**Comment by DanielnetoDotCom** [comment]
![chrome-capture-2023-10-8](https://github.com/WWBN/AVideo/assets/18009293/d611d09e-2d63-46e7-8bf2-c77415620773)

**Comment by para2022** [comment]
And have your refreshed the page a few times like i said, in brave or firefox? If you visit the page for the first time it works i have already gone over this many times. It works in MAC safari, doesnt work in iphone, brave or firefox and i havent tested other browsers. It works after i clear the cache, but only until i refresh the homepape a few times and it comes back. If i turn on the big video on homepage with carosel and animated gif, only then will it work and continue to work. It does not affect the suggest group at the top of the page only anything under that group. It is buggy, its as simple as that. I have told you when this problem started and i can see the updated files in the gallery plugin that were recently updated. Thats when it all started and the sort issue i have also. I had no problems before that update. I cannot be bothered now its took 2 days of life and i know for a fact it isnt my setup.

**Comment by para2022** [comment]
[recording (5).webm](https://github.com/WWBN/AVideo/assets/107706817/a3409aa8-ece6-4b15-895b-3b18014e45d0)

**Comment by DanielnetoDotCom** [comment]
can you please check this update?

**Comment by para2022** [comment]
sorry still the same and same error in dev tools

**Comment by para2022** [comment]
![Uploading Screenshot 2023-11-09 at 15.56.49.png…]()

**Comment by DanielnetoDotCom** [comment]
did you clear the cache?

**Comment by para2022** [comment]
yea it always works after a cache clear. But only until i refresh the page and then it stops working again. So long as i stay on the homepage after a cache reset the load more works perfectly until i refresh then it stops.

**Comment by para2022** [comment]
Could it be the cache function causing it? After the cache next updates then it breaks.

1. When I clear the cache, the load more on homepage works.
2. If i refresh and the page changes or when the cache updates next i.e 1 minute then it will stop working again.
3. I see also the 'clear first page cache' function doesn't work, only the 'clear cache directory' works

**Comment by DanielnetoDotCom** [comment]
can you please try this update?

**Comment by para2022** [comment]
i am missing the top navbar now so their are no controls for the site.

https://ourtube.co.uk/

**Comment by DanielnetoDotCom** [comment]
Sorry is the top bar fine?

Is the load more fine?

**Comment by para2022** [comment]
Ignore the nav bar comment it’s fine. The load more is behaving properly now on the homepage. Also the other issues first mentioned in this thread are also resolved. Thanks.

**Comment by DanielnetoDotCom** [comment]
your navbar is not loading for me

**Comment by DanielnetoDotCom** [comment]
NVM, it just load, I am not sure why

**Comment by para2022** [comment]
Yes it is actually behaving strangely. I just noticed it on my iPhone

**Comment by DanielnetoDotCom** [comment]
I guess is some CSS that was not properly updated. just keep the eyes opened. I will send some update in case it happens again I will be able to debug

**Comment by para2022** [comment]
When i click on a video the navbar vanishes

**Comment by DanielnetoDotCom** [comment]
Got it, please try this

**Comment by para2022** [comment]
Fab. All working correctly now. Thanks again :)

**Comment by para2022** [comment]
The infinite scroll is left on with the new changes dan (not the homepage). https://ourtube.co.uk/cat/music 
You can see that if i scroll to the bottom i cant use the numbered block or load more, as the infinite scroll will activate. Can you turn it off so that we can get to the 'load more' or numbered block to use those functions. Thanks

**Comment by DanielnetoDotCom** [comment]
what do you mean, can you please screenshot?

**Comment by DanielnetoDotCom** [comment]
NVM, I see it is autoloading. hold on

**Comment by DanielnetoDotCom** [comment]
Please try now

**Comment by para2022** [comment]
All working beautifully now. Thanks :)
